src/plugins/messenger/MessengerPreview/components/MessengerGenericTemplate/MessengerGenericTemplate.tsx

```diff
@@ -240,6 +244,7 @@ export const getMessengerGenericTemplate = ({
 											button={button}
 											onClick={e => onAction(e, button)}
 											className="webchat-carousel-template-button"
+                                            config={this.props.config}
```

Comment:
This particular line is indented using spaces, but the wrapper element is indented using Tabs. We need to find a solution for this. We could see the same issue in other PRs too.